### ISSUE TITLE
ERROR com.amazon.alexa.avs.AVSApp - An error occured creating speech reques

### ISSUE THREAD

**Issue by simons9989** [amzn/alexa-avs-raspberry-pi#180]
when I press the 'Start Listening' button, it appear one error java pop up with no text, just OK button. And then I click the button, it looks like working normally moving a process bar parallelly but back at console, I can see the errors like this

 [AWT-EventQueue-0] ERROR com.amazon.alexa.avs.AVSApp - An error occurred creating speech request java.lang.NullPointerException at com.amazon.alexa.avs.AudioCapture.startCapture(AudioCapture.java:74) ~[classes/:?]
	at com.amazon.alexa.avs.AudioCapture.getAudioInputStrea ...

I geuss that it may causes the errors to try to use just earphones as a microphone which comes from iPhone,  but I don't know why exactly.

Anyone can help me and have a same experience? Please any ideas to resolve it.

Thanks you

**Comment by simons9989** [comment]
I think I've done it!
I know of two things, first I put my earphones in 3.5 mm jack but I ends up know this is for stereo audio & video composite, Once again I would say I've got to have usb microphone.
And second, I put the code in AVSController to get logs and I got null points as a "microphone"
As a result, I'm pretty sure that no microphone causes the errors!
I hope i would be helpful for anyone hardly not to solve this problem
